apps/fishing-map/features/map/error-notification/error-notification.slice.ts

```diff
@@ -0,0 +1,49 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit'
```

Comment:
as there are three slices doing almost the same, WDYT about merging annotations, rules and this to a new `map-controls.slice` ?